### ISSUE TITLE
Services crashes while we use HEIC image format in docker container.

### ISSUE THREAD

**Issue by BhushanShah** [dlemstra/Magick.NET#318]
### Prerequisites

- [x] I have written a descriptive issue title
- [X] I have verified that I am using the latest version of Magick.NET
- [X] I have searched [open](https://github.com/dlemstra/Magick.NET/issues) and [closed](https://github.com/dlemstra/Magick.NET/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description

I have created a dotnet core application which runs in docker container. Whenever I use HEIC image, it works locally in windows but when I deploy the same in test environment, it crashes the application. And I receive a below error:

`dotnet: symbol lookup error: /app/runtimes/linux-x64/native/Magick.NET-Q16-x64.Native.dll.so: undefined symbol: de265_init`

Below is the code where it crashes:

```
MagickReadSettings readSettings = new MagickReadSettings{Format = MagickFormat.Heic};
MagickImage objImage = new MagickImage(file.OpenReadStream(), readSettings);
```

### System Configuration

- Magick.NET version: 7.7.0
- Environment (Operating system, version and so on): Linux (docker)
- Additional information:

**Comment by dlemstra** [comment]
Can you reproduce your issue when you use the `.dll.so` file from this development build: https://dev.azure.com/dlemstra/Magick.NET/Magick.NET%20Team/_build/results?buildId=619? And what is your docker base image?

**Comment by BhushanShah** [comment]
Hi,

I am using microsoft/aspnetcore-build as a docker base image.  
I will try using **imageMagick 7.0.8-12** beta version and will let you know.

Thanks,
Bhushan Shah

**Comment by BhushanShah** [comment]
Sure, I will try that and will update here.

**Comment by BhushanShah** [comment]
Hi,

Still getting the same error with latest version 7.8.0.

`dotnet: symbol lookup error: /app/runtimes/linux-x64/native/Magick.NET-Q16-x64.Native.dll.so: undefined symbol: de265_init`

**Comment by dlemstra** [comment]
Okay thanks for trying and the info. Will give it a try later this week.

**Comment by dlemstra** [comment]
I can reproduce your issue. But now i need to figure out why this fails... Will get back to you when I have more info.

**Comment by BhushanShah** [comment]
Thanks for the update. Please let me know once you reach some conclusion or you release a fix.

**Comment by dlemstra** [comment]
It appears the linking order was incorrect. I just pushed a patch for this and I can now read HEIF files with the native library from this build: https://dev.azure.com/dlemstra/Magick.NET/_build/results?buildId=631. Can you also give it a try?

**Comment by BhushanShah** [comment]
Sure. I will give a try and will update you.

**Comment by BhushanShah** [comment]
Hi,

Can you please release a patch to nuget so that I can use that version? That's is the only way for me to update a build on the server.

Thanks,
Bhushan Shah

**Comment by dlemstra** [comment]
I am working on some other changes. When they are done I will publish a new release.

**Comment by BhushanShah** [comment]
Thanks for the update. Appreciate your support.

**Comment by BhushanShah** [comment]
HI Dirk,

Just wanted to check when you are planning to release the fix?

Thanks,
Bhushan Shah

**Comment by dlemstra** [comment]
TL;DR When it's done 👍 

I hope to publish the new release within the next few weeks. I really don't know exactly when I will publish the new release. I will try to do it when I am done with the changes I am working on. But I don't know when that will be done.

**Comment by dlemstra** [comment]
I just publish Magick.NET 7.9.0.0 that includes a fix for your issue.

**Comment by BhushanShah** [comment]
Thank you, @dlemstra.

It is working fine after upgrading it to 7.9.0.0.
